testnet/stacks-node/src/node.rs

```diff
@@ -550,9 +544,8 @@ impl Node {
             }
             tx.commit().unwrap();
         }
-        let atlas_config = AtlasConfig::default(false);
-        let atlasdb =
-            AtlasDB::connect(atlas_config, &self.config.get_atlas_db_file_path(), true).unwrap();
+        let _atlas_config = Self::make_atlas_config();
```

Comment:
Is this used somewhere?

Reply:
Deleted in https://github.com/stacks-network/stacks-blockchain/pull/3086/commits/cb9f8dca0f9697e7cfe04c1b2e776de4b8aec8ce